x-pack/plugins/enterprise_search/public/applications/app_search/utils/role/has_scoped_engines.test.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { roleHasScopedEngines } from './';
+
+describe('roleHasScopedEngines()', () => {
```

Comment:
*do not* make this update for this PR, it's not worth running CI. Just noting that we usually don't add parens to the end of function names in describes.

```suggestion
describe('roleHasScopedEngines', () => {
```

Reply:
haha thanks Jason, I was totally about to smash commit suggestion before seeing the _do not_ (italics totally worked). This is an accidental copypaste carry over from https://github.com/elastic/kibana/blob/19691040105be7270fd4c7129f2f0c619c8503a1/x-pack/plugins/enterprise_search/public/applications/app_search/utils/role/get_role_abilities.test.ts#L54
- I think I originally added the `()` because it's a function helper within a 'class'/obj that has static vars mixed in with fn's 🤔 not sure if the `()` is helpful in those scenarios - agreed they're not helpful for utility files where everything is a function

Reply:
@scottybollinger if you're down, feel free to make Jason's suggested copy change in a future role mappings PR!